docs/source/execute_api.rst

```diff
@@ -0,0 +1,143 @@
+Executing notebooks
+===================
```

Comment:
I agree with @takluyver here. In most of my use cases I need to execute notebooks and save the results as notebook (using `nbformat`, saving is not even handled by nbconvert).

Saving as notebook has 2 advantages for me. One is that the file can be distributed and is "executable". Compare this to a static representation in a sphinx documentation, that is harder to re-run (I should add a link to the original unexecuted notebook, but it's nice to have the output). Second, for debugging, it's easier to start from a notebook with all the output and re-execute it step by step, instead of starting from a  clean (no output) notebook. In this latter case, I use nbconvert for batch execution of a set of notebooks and when something fails I dig the problem by opening the failing notebook (containing output).